reana_client/cli/analyses.py

```diff
@@ -61,7 +62,7 @@ def analysis_validate(ctx, file):
     except Exception as e:
         logging.info('Something went wrong when trying to validate {0}'
                      .format(click.format_filename(file)))
-        logging.debug(str(e))
+        logging.error(traceback.print_exc())
```

Comment:
`traceback.format_exc()` here and elsewhere?

Reply:
force pushed the commit with `print_exc()`